efaar_benchmarking/data_loading.py

```diff
@@ -1,17 +1,98 @@
 import os
+import shutil
+import tempfile
+import zipfile
+from concurrent.futures import ThreadPoolExecutor, as_completed
 
 import numpy as np
+import pandas as pd
 import scanpy as sc
 import wget
+from sklearn.covariance import EllipticEnvelope
+
+CP_FEATURE_FORMATTER = (
+    "s3://cellpainting-gallery/cpg0016-jump/"
+    "{Metadata_Source}/workspace/profiles/"
+    "{Metadata_Batch}/{Metadata_Plate}/{Metadata_Plate}.parquet"
+)
+
+
+def load_cpg16_crispr(data_path="data/", filter_by_intensity=True, filter_by_cell_count=True, drop_image_cols=True):
+    plate_file_name = "plate.csv.gz"
+    well_file_name = "well.csv.gz"
+    crispr_file_name = "crispr.csv.gz"
+    plate_file_path = os.path.join(data_path, plate_file_name)
+    well_file_path = os.path.join(data_path, well_file_name)
+    crispr_file_path = os.path.join(data_path, crispr_file_name)
+    if not (os.path.exists(plate_file_path) and os.path.exists(well_file_path) and os.path.exists(crispr_file_path)):
+        source_path = "https://zenodo.org/records/7661296/files/jump-cellpainting/datasets-v0.5.0.zip?download=1"
```

Comment:
where does this come from?

Reply:
I believe this is where you downloaded the three csv files you put in the repo, no? I want to have everything downloaded from scratch.